src/platform/packages/shared/kbn-ebt-tools/src/performance_metrics/context/measure_interaction/index.ts

```diff
@@ -33,24 +33,25 @@ export function measureInteraction(pathname: string) {
      * @param customMetrics - Custom metrics to be included in the performance measure.
      */
     pageReady(eventData?: EventData) {
-      let performanceMeta: PerformanceMeta | undefined;
+      const performanceMeta: PerformanceMeta = {};
       performance.mark(perfomanceMarkers.endPageReady);
 
-      if (eventData?.meta) {
-        const { rangeFrom, rangeTo, description } = eventData.meta;
+      if (eventData?.meta?.rangeFrom && eventData?.meta?.rangeTo) {
+        const { rangeFrom, rangeTo } = eventData.meta;
 
         // Convert the date range  to epoch timestamps (in milliseconds)
         const dateRangesInEpoch = getDateRange({
           from: rangeFrom,
           to: rangeTo,
         });
 
-        performanceMeta = {
-          queryRangeSecs: getTimeDifferenceInSeconds(dateRangesInEpoch),
-          queryOffsetSecs:
-            rangeTo === 'now' ? 0 : getOffsetFromNowInSeconds(dateRangesInEpoch.endDate),
-          description,
-        };
+        performanceMeta.queryRangeSecs = getTimeDifferenceInSeconds(dateRangesInEpoch);
+        performanceMeta.queryOffsetSecs =
+          rangeTo === 'now' ? 0 : getOffsetFromNowInSeconds(dateRangesInEpoch.endDate);
```

Comment:
Based on the diff I guess we don't need to add a new test in `measure_interaction.test.ts` right?